repaygithub/cactus
# Design 196 index theme pages

[DESIGN-196](https://repayonline.atlassian.net/browse/DESIGN-196)

Added the section for **Design Guides** to the docs website and the [Theme](https://www.figma.com/proto/J9TeJ5homSkfSwV1sCFLoF/REPAY-STYLEGUIDE-V-1?node-id=4722%3A0&viewport=-3135%2C461%2C0.5077165961265564&scaling=min-zoom) section from the style guide.

### Test

- Open the **Design Guides** section on the website
- See that all the info from [Repay Theme](https://www.figma.com/proto/J9TeJ5homSkfSwV1sCFLoF/REPAY-STYLEGUIDE-V-1?node-id=4722%3A0&viewport=-3135%2C461%2C0.5077165961265564&scaling=min-zoom) section is on the **Theme** page inside ***Design Guides** section.
